.travis.yml

```diff
@@ -7,6 +7,74 @@ env:
 
 matrix:
   include:
+    - name: macOS 10.13, llvm 7
+      env: CC=/usr/local/opt/llvm@7/bin/clang
+      env: CXX=/usr/local/opt/llvm@7/bin/clang++
+      os: osx
+      osx_image: xcode10.1
+      sudo: false
+      before_install:
+        - brew update
+        - brew install llvm@7 libomp
+        - if brew outdated | grep -q python; then brew upgrade python; fi
+
+    - name: macOS 10.13, GCC 8
+      env: CC=/usr/local/opt/gcc@8/bin/gcc-8
+      env: CXX=/usr/local/opt/gcc@8/bin/g++-8
+      os: osx
+      osx_image: xcode10.1
+      sudo: false
+      before_install:
+        - brew update
+        - brew install libomp
```

Comment:
In this image gcc-8 is preinstalled. However, this may lead to errors if one just updates the xcode image without checking the gcc version. In c0f5510f43b03bd98d1d3af9f7b72731937afcaf I am using the homebrew addon as you suggested and added gcc-8 to the packages to be installed.